spec/factories/case_log.rb

```diff
@@ -4,7 +4,6 @@
     managing_organisation { FactoryBot.create(:organisation) }
     trait :about_completed do
       gdpr_acceptance { "Yes" }
-      sale_or_letting { "Letting" }
```

Comment:
Not sure if this is correct, as `sale_or_letting` might still be required for bulk uploads?

Reply:
Fine for now I think